lib/brcobranca/boleto/credisis.rb

```diff
@@ -91,9 +101,29 @@ def agencia_conta_boleto
       end
 
       # Segunda parte do código de barras.
-      # @return [String] 25 caracteres numéricos.
+      #
+      # 01 a 03 03 Código da IF “097”
+      # 04 a 04 01 Código da Moeda “9” (Real)
+      # 05 a 05 01 *Dígito Verificador (DV)
+      # 06 a 09 04 **Fator de Vencimento
+      # 10 a 19 10 Valor
+      # 20 a 44 25 ***Campo Livre (Nosso Número)
+      #
+      # MONTAGEM DO CÓDIGO DE BARRAS:
+      # 1     2   3   4     5            6        7
+      # AAA   B   K   UUUU  VVVVVVVVVV   CCCCC    DDDDDDDDDDEEEEEEEEEE
+      #
+      # 1. - Número do Banco: “097”
+      # 2. - Moeda: “9”
+      # 3. - DV do Código de Barras, Baseado no Módulo 11 (Vide Anexo X).
+      # 4. - Fator de Vencimento do Boleto (Vide Anexo VII).
+      # 5. - Valor do Título, expresso em Reais, com 02 casas decimais.
+      # 6. - Fixo Zeros: Campo com preenchimento Zerado “00000”
+      # 7. - Composição do Nosso Número: 097XAAAACCCCCCSSSSSS
+      #
+      # @return [String] 25 caracteres numéricos..
       def codigo_barras_segunda_parte
-        "00#{convenio}#{codigo_cedente}#{numero}#{carteira}".rjust(25, '0')
+        "#{'0' * 5}#{nosso_numero_boleto}".rjust(25, '0')
```

Comment:
Nas primeiras 5 posições é para preencher com zeros mesmo?  Código da IF, Moeda, etc não são necessários? 🤔

Reply:
Aaah, essa parte do código da IF, Moeda não faz parte dessa segunda parte do código de barras, é isso? https://github.com/monde-sistemas/brcobranca/pull/152/commits/ade25e721a247f4641583e5d86b84759b9018707#diff-632d287cd25c2046cdcd6909b1d5f312R103

Se for, acho que poderia remover o comentário acima desse método que não se refere a segunda parte do código de barras 🤔

Reply:
Isso mesmo, deixei apenas a parte referente a segunda parte;